Danger/Dangerfile.swift

```diff
@@ -493,47 +28,18 @@ private extension SwiftLint.SwiftlintPath {
     
 }
 
-// MARK: - Check routine
+// MARK: - チェックルーチン
 let danger = Danger()
 
-// If there's any modification in bootstrap.sh file, ask the reviewer to check if he needs to update Bitrise workflows.
-if danger.hasBrewfileBeenModified {
-    markdown("- [ ] Check bitrise workflow")
-    warn("There's modification in bootstrap.sh file. Please remember to check if needed to update Bitrise workflow.")
-}
-
-// If the head branch refers to a GitHub issue, comment it in the PR page.
-if let githubIssue = danger.githubIssue {
-    message("Resolve #\(githubIssue)")
-}
-
-// SwiftLint format check.
+// SwiftLint のワーニング等確認
 SwiftLint.lint(.modifiedAndCreatedFiles(directory: nil), inline: true, configFile: "swiftlint.yml", swiftlintPath: .installedBySwiftPM)
 
-// Xcode summary warnings check.
+// Xcode ビルドのワーニング等確認
 danger.kantoku.parseXCResultFile(at: ProcessInfo.xcTestResultPath, configuration: .default)
 
 // Xcode test coverage check.
 // temporarily disable this because hasn't found a way to get coverage at post_xcodebuild yet
 //Coverage.xcodeBuildCoverage(.xcresultBundle(ProcessInfo.xcTestResultPath), minimumCoverage: 60)
```

Comment:
<!--
  0 failure: 
  1 warning:  Prefer at least o...
  
  
  DangerID: danger-id-Danger;
  File: Danger/Dangerfile.swift;
  Line: 42;
-->

- :warning: Prefer at least one space after slashes for comments. (`comment_spacing`)